HIP/hip-1084.md

```diff
@@ -0,0 +1,118 @@
+might send malformed transactions to the network, where there would be no one to pay for the processing of those transactions.
+This fee is unnecessary for successful transactions since the intrinsic gas fee covers the base costs (gossip & consensus).
+Therefore, this HIP proposes that the fee for submitting `EthereumTransactions` to the network should only be charged if the
+JSON-RPC relay submits a transaction that fails due diligence checks, such that the EOA cannot be charged for the overhead
+of gossip and consensus.
+
+## User stories
+
+1. As a relay operator, I want to submit an `EthereumTransaction` on behalf of an EOA without paying additional fees.
+2. As a node operator, I want to charge relays that submit transactions that cannot be charged to the EOA.
+3. As a user, I want to use the same pricing structure and have the same user experience for Hedera relays as with other
+   networks.
+  
+## Specification
+
+Definitions:
+ - `evm tx`: The RLP encoded transaction bytes defined by the Ethereum yellow paper that form a standard Ethereum transaction
+ - `relay`: A system that, given an `evm tx`, wraps it in an HAPI `EthereumTransaction` and submits it to Hedera.
+            This is also the `payer` for the `EthereumTransaction`. See
+            [TransactionBody.transactionID](https://github.com/hashgraph/hedera-protobufs/blob/main/services/transaction_body.proto#L104)
+ - `EOA`: The account that created and signed the `evm tx` and submitted it to a `relay`. This is the `from` in the
+          [Ethereum Transaction Schema](https://github.com/ethereum/execution-apis/blob/main/src/eth/transaction.yaml)
+          often noted as (tx.from)
+
+### Smart Contract Creation and Execution Transactions
+
+The `relay` will **only** be charged if handling of the `EthereumTransaction` fails **before** any bytecodes are executed.
+In other words, the `relay` is charged when one of the pre-requisite checks for `EthereumTransaction` fails, regardless of
+the execution status of the transaction itself once it begins smart contract creation or execution in the EVM.
+
+Examples:
+ - The `relay` is charged if the `evm tx` cannot be parsed or is otherwise malformed
+ - The `relay` is charged if the `EOA` specified in the `evm tx` refers to a non-existent account
+ - The `relay` is charged if the `evm tx` was not properly signed by the keys of the `EOA`
+ - The `relay` is charged if the `EOA` has insufficient funds to pay the intrinsic gas fee
+ - The `relay` is charged if the `nonce` check fails
+ - The `relay` is not charged if the contract is called but concludes with an error code 
+ - The `relay` is not charged if the contract is called and concludes without error
+
+In summary the `relay` account is not charged if the `EthereumTransaction` transaction completes in a 
+`CONTRACT_REVERT_EXECUTED` or `SUCCESS` HAPI response code, all other responses codes will result in a charge to the
+relay account.
```

Comment:
If such a pre-check fails on consensus node side, how it will be made sure the relay has enough balance to be charged for the submission of a faulty transaction?

Reply:
This is standard logic in todays network where precheck varies a HAPI submitters balance to process a transaction before gossiping out.
So there's no change here